lighthouse-core/test/report/report-test.js

```diff
@@ -78,15 +81,11 @@ describe('Report', () => {
     assert.ok(html.includes('&quot;lighthouseVersion'), 'lhresults were escaped');
     assert.ok(/Version: x\.x\.x/g.test(html), 'Version doesn\'t appear in report');
     assert.ok(html.includes('export-button'), 'page includes export button');
-
-    assert.ok(html.includes('printButton = document.querySelector'),
-                            'print button functionality attached');
-    assert.ok(html.includes('openButton = document.querySelector'),
-                            'open button functionality attached');
     assert.ok(html.includes('share js-share'), 'has share button');
```

Comment:
i guess it'd be a pain to assert that the share icon has display:none in this case, yeah?

ah well.